prow/plugins/override/override_test.go

```diff
@@ -131,16 +131,6 @@ func (c *fakeClient) presubmits(_, _ string, _ config.RefGetter, _ string) ([]co
 }
 
 func (c *fakeClient) CreateComment(org, repo string, number int, comment string) error {
-	switch {
-	case org != fakeOrg:
```

Comment:
Why are you regressing this?

Reply:
`comment on get pr failure` test is testing a failure for getting a pr with a wrong PR number.
In case having those validations on `CreateComment` function then it would fail as well for having a wrong PR number.

Since the override tests don't test creating comments by any test, but utilize them for testing others, I figured this shouldn't have any special treatment at all and the `fakeClient` would just create the comments as expected (for later being verified as this PR fixes)